convex/getEvents.ts

```diff
@@ -39,8 +36,8 @@ export default authenticatedQuery(async ({ db }, user) => {
       return {
         ...event,
         hostDetails: {
-          name: host.name,
-          isCurrentUser: host._id.equals(user._id),
+          name: host?.name,
```

Comment:
The `host` is possibly null because it uses `db.get()`. Every event really should have a host, so while this works in practice I don't love using optional chaining here.  I suppose `host!.name` could be used instead to tell TypeScript we're certain it will be set.

Maybe we should just be throwing an error up above is the host can't be found?

Reply:
Yeah, that sounds good to me!